client/app/components/contact-form.js

```diff
@@ -0,0 +1,43 @@
+import Ember from 'ember';
+import config from '../config/environment';
+const isTest = config.APP.STUB_EXTERNAL_MOCK;
+
+export default Ember.Component.extend({
+
+  districtContactEmail: '',
+  districtContactSubmitted: false,
+  districtContactErrored: false,
+
+  getSubmittedName: Ember.computed('name', function() {
+    return this.get('name') != undefined ? `${this.get('name')} ,` : '';
+  }),
+
+  actions: {
+    submitDistrictContact() {
+      this.set('districtContactErrored', false);
+
+      if (Ember.isPresent(this.get('districtContactEmail'))) {
+
+        Ember.$.ajax({
+          url: 'https://formspree.io/schoolbot-contact@vermonster.com',
+          method: 'POST',
+          dataType: 'json',
+          data: {
+            _replyto: this.get('districtContactEmail'),
+            _subject: `District contact by ${this.get('districtContactEmail')}`,
+            name: this.get('name'),
+            message: this.get('message')
+          }
+        }).then(() => {
+          this.set('districtContactSubmitted', true);
+        }).catch(() => {
+          if (isTest) {
```

Comment:
OK, I think your solution with the initializer is a good one in terms of avoiding the entanglement of test code in the app (though it might require us to include Mockjax in the production build, which is unfortunate). Perhaps we can add a TODO comment to look into puffing-billy later.